Streetcode/Streetcode.WebApi/Controllers/AdditionalContent/TagController.cs

```diff
@@ -35,6 +37,9 @@ public async Task<IActionResult> GetTagByTitle([FromRoute] string title)
     }
 
     [HttpPost]
+    [Authorize(Roles = nameof(UserRole.Admin))]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
```

Comment:
Can we make it just   [Authorize(Roles = nameof(UserRole.Admin))] ?
As I was implementing authorization on my last project using .NET I used only the decorator, and it worked, so in .NET it's possible to use just one, you just need to figure out how.